lib/watir-webdriver/checkers.rb

```diff
@@ -0,0 +1,88 @@
+module Watir
+  class Checkers
+
```

Comment:
I'd include `Enumerable` here and implement `#each` to make blocks iterable so we'd give more options to the users without adding much of the complexity.